corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -190,7 +190,7 @@ def get_all_instances_referenced_in_xpaths(domain, xpaths):
     instances = set()
     unknown_instance_ids = set()
     for xpath in xpaths:
-        instance_names = re.findall(instance_re, xpath)
+        instance_names = re.findall(instance_re, xpath, re.UNICODE)
```

Comment:
Allows alphanumeric unicode for `\w` https://docs.python.org/2/library/re.html#re.U